doc/api/esm.md

```diff
@@ -124,12 +124,12 @@ as ES modules and `.cjs` files are always treated as CommonJS.
 ### Package scope and file extensions
 
 A folder containing a `package.json` file, and all subfolders below that folder
-down until the next folder containing another `package.json`, is considered a
-_package scope_. The `"type"` field defines how `.js` files should be treated
-within a particular `package.json` file’s package scope. Every package in a
+until the next folder containing another `package.json`, are a
+_package scope_. The `"type"` field defines how to treat `.js` files
+within the package scope. Every package in a
 project’s `node_modules` folder contains its own `package.json` file, so each
-project’s dependencies have their own package scopes. A `package.json` lacking a
-`"type"` field is treated as if it contained `"type": "commonjs"`.
+project’s dependencies have their own package scopes. If a `package.json` file
+does not have a `"type"` field, the default `"type"` is `"commonjs"`.
```

Comment:
Is it worth noting that invalid values are also interpreted as `"commonjs"`?
```suggestion
does not have a `"type"` field or an invalid one, the default `"type"` is
`"commonjs"`.
```

Reply:
If that's the case, then that seems like a good addition. I'd prefer that be a different pull request so that this one is kept purely style and grammar.